var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -104,13 +98,28 @@ def cmake_args(self):
         else:
             options.append('-DGMX_GPU:BOOL=OFF')
 
-        simd_value = self.spec.variants['simd'].value
-        if simd_value == 'auto':
-            pass
-        elif simd_value == 'none':
+        simd_features = {
+            'sse2': 'SSE2',
+            'sse4_1': 'SSE4.1',
+            'avx128fma': 'AVX_128_FMA',
+            'avx256': 'AVX_256',
+            'axv128': 'AVX2_128',
+            'avx2_256': 'AVX2_256',
+            'avx512': 'AVX_512',
+            'avx512knl': 'AVX_512_KNL',
+            'qpx': 'IBM_QPX',
+            'ace': 'Sparc64_HPC_ACE',
+            'vmx': 'IBM_VMX',
+            'vsx': 'IBM_VSX',
+            'neon': 'ARM_NEON',
+            'neon_asimd': 'ARM_NEON_ASIMD'
```

Comment:
We don't have support for `neon_asimd`. Can you link some reference to check how to add this too?

Reply:
about INSTALL in gromacs@2019.4:
- "ARM_NEON" 32-bit ARMv7 with NEON support.
- "ARM_NEON_ASIMD" 64-bit ARMv8 and later.

I think ARM_NEON_ASIMD is neon.

Reply:
Thanks @tkameyama

Reply:
Ok, I dropped support for `ARM_NEON` for now.